package.json

```diff
@@ -10,8 +10,6 @@
     "serve-vue": "vue-cli-service serve --open --host localhost"
   },
   "dependencies": {
-    "ajv": "8.11.0",
-    "ajv-keywords": "5.1.0",
```

Comment:
@pauline2k - I don't think the `ajv` dependencies (above) are used. Do you know why they were added in the first place? The purpose of the library is front-end schema/object validation: https://ajv.js.org/guide/getting-started.html

Reply:
@johncrossman `ajv` and `ajv-keywords` are deeply nested dependencies of other packages, and in the past we've found ourselves having to lock down versions in order to avoid other npm problems. If we can actually do without them in the package.json, that is absolutely all to the good.